### ISSUE TITLE
Missing customized +snippet tags

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#795]
```
See http://www.google.com/webmasters/+1/button/ for customized attributes and 
tags for +Snippets.
```

Original issue reported on code.google.com by `sethladd@google.com` on 1 Feb 2012 at 3:04

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 1 Feb 2012 at 4:35
- Changed state: **Assigned**
- Added labels: **Milestone-Q12012**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision f1bad30c2b2a.
```

Original comment by `ericbide...@html5rocks.com` on 6 Feb 2012 at 10:36
- Changed state: **Fixed**
